_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,145 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+### Expectation
+
+Each target element is [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by an element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has a [semantic role](#semantic-role) matching one of the [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+## Assumptions
+
+_There are currently no assumptions_
+
+## Accessibility Support
+
+When a required owned element is not a child, but rather a descendant, certain AT have issues recognizing the owned element.
```

Comment:
Is it correct to say:
[[
This rule relies on assistive technologies that can recognize the owned element when it is a nested descendant, rather than an immediate child.
]]

Just an attempt at making it more specific and less open to interpretation of what "issues" are.

Reply:
I don't think this is a note about assistive technologies. AT aren't supposed to recognise items any level deep as part of the same context.